clover_blocks/CMakeLists.txt

```diff
@@ -0,0 +1,81 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(clover_blocks)
+
+find_package(catkin REQUIRED COMPONENTS message_generation)
+
+add_message_files(
+  FILES
+  Prompt.msg
+)
+
+add_service_files(
+  FILES
+  Run.srv
+)
+
+generate_messages(
+  DEPENDENCIES
+  # std_msgs  # Or other packages containing msgs
+)
+
+catkin_package(
+#  INCLUDE_DIRS include
+#  LIBRARIES roslaunch_editor
+#  CATKIN_DEPENDS other_catkin_pkg
```

Comment:
`CATKIN_DEPENDS` should not be commented and should have `message_runtime` as a dependency.